packages/modules/web-viewer-react/src/services/ItwinViewer.ts

```diff
@@ -66,23 +70,25 @@ export class ItwinViewer {
       throw new Error("Please provide a valid iTwinId and iModelId");
     }
 
+    const viewerProps: WebProps = {
+      authClient: this.authClient,
+      iTwinId: args?.iTwinId ?? "",
+      iModelId: args?.iModelId ?? "",
+      changeSetId: args?.changeSetId,
+      uiConfig: this.uiConfig,
+      appInsightsKey: this.appInsightsKey,
+      onIModelConnected: this.onIModelConnected,
+      frontstages: this.frontstages,
+      viewportOptions: this.viewportOptions,
+      uiProviders: this.uiProviders,
+      theme: this.theme,
+      enablePerformanceMonitors: this.enablePerformanceMonitors,
+    } as WebProps;
+
     // render the viewer for the given iModel on the given element
-    ReactDOM.render(
-      React.createElement(Viewer, {
-        authClient: this.authClient,
-        iTwinId: args?.iTwinId,
-        iModelId: args?.iModelId,
-        changeSetId: args?.changeSetId,
-        uiConfig: this.uiConfig,
-        appInsightsKey: this.appInsightsKey,
-        onIModelConnected: this.onIModelConnected,
-        frontstages: this.frontstages,
-        viewportOptions: this.viewportOptions,
-        uiProviders: this.uiProviders,
-        theme: this.theme,
-        enablePerformanceMonitors: this.enablePerformanceMonitors,
-      } as WebViewerProps),
-      document.getElementById(this.elementId)
-    );
+    // ReactDOM.render(
```

Comment:
shouldnt this be un-commented? We'll need to render something :-)

Reply:
For some reason it was complaining about one of the args being undefined, need to spend more time looking at this;

iirc, this was only used by the "cdn viewer" and i think we need to re-discuss this; was hoping we can remove this, really doesnt feel like this is something we should be delivering

Reply:
this is the only thing left to do in the pr i think; im in favor of removing this entirely; but we can take the discussion offline

Reply:
this is the CDN viewer. We can't remove it without a suitable replacement

Reply:
yea, but have we even published an updated cdn viewer in a yr? is anyone actually using the cdn viewer?

Reply:
no and maybe :-). It's still being marketed to potential users though. We'll need to support it still (at least for the short term). It's more on me not being able to prioritize updating the pipelines why it hasnt been deployed

Reply:
I cant wait to delete the cdn viewer :) I'll try to survive until then.

Updated the class